universe/UniverseGenerator.cpp

```diff
@@ -872,13 +872,12 @@ void InitEmpires(const std::map<int, PlayerSetupData>& player_setup_data)
 
     // create empire objects and do some basic initilization for each player
     for (const auto& entry : player_setup_data) {
-        int         player_id =     entry.first;
-        if (player_id == Networking::INVALID_PLAYER_ID)
-            ErrorLogger() << "InitEmpires player id (" << player_id << ") is invalid";
+        int         empire_id =     entry.first;
+        if (empire_id == ALL_EMPIRES)
+            ErrorLogger() << "InitEmpires empire id (" << empire_id << ") is invalid";
 
-        // use player ID for empire ID so that the calling code can get the
-        // correct empire for each player ID  in player_setup_data
-        int         empire_id =     player_id;
+        // use map key for empire ID so that the calling code can get the
```

Comment:
Should this comment be before the line above that assigns `empire_id`?

Reply:
Fixed.